### ISSUE TITLE
[a11y]: Radio tiles inside a side panel don't seem to scroll into viewport when focused

### ISSUE THREAD

**Issue by szamoca-at-the-corp** [carbon-design-system/carbon#20851]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

v1.81.0

### React version

v18.3.1

### Automated testing tool and ruleset

Keyboard_Firefox

### Description

**Origin**
In our application, the AVT team discovered an issue regarding radio tiles that are rendered inside a side panel.

> Component: Reflow
>
> Issue description: Focused radio buttons are not appearing on the screen on 'Side panel name' panel when screen resized to 400%.
>
> Recommended fix: Focused radio buttons should appear on the screen on 'Side panel name' panel when screen resized to 400%.
>
> User impact: If a user is not able to access or see the content properly when the screen is resized to 400%, it can impact users with visual impairments or other disabilities that require larger text sizes to read and understand the content. This can make it difficult for these users to read and understand the content, leading to exclusion, frustration, and a negative user experience.
>
> WCAG Mapping: 1.4.10 Reflow
>
> Level of Compliance: AA
>
> Severity: Medium
>
> Test tool: Keyboard_Firefox

**Further context**
The problematic area is a tile group with radio tiles inside, that itself is inside a side panel. Regardless of zoom level, if the section is out of the viewport, it seems like the radio tiles receive focus when tabbed, however they are not getting scrolled into the viewport. Behaviour continues when selecting via <kbd>up</kbd>/<kbd>down</kbd> keys too. Other types of inputs (TextInput for example) rendered in the same side panel are scrolled to correctly. This behaviour is observable in other places with similar setups as well (side panel + radio tile).

### WCAG 2.1 Violation

Success Criterion 1.4.10 Reflow

### Reproduction/example

https://stackblitz.com/edit/github-xl8vex7v?file=src%2FExample%2FChildrenContent.jsx

### Steps to reproduce

1. Open side panel
2. By default you'll be focused on the first interactive element
3. Align/zoom side panel content in such a way the radio tile group is out of the viewport
4. Start tabbing through the interactive elements/inputs
5. Notice when getting to the radio tile section, page does not scroll the elements into the viewport
6. Same behaviour continues when selecting different radio tiles via up/down keys

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by Gururajj77** [comment]
Thanks for raising this issue !! I confirm this issue exist.
